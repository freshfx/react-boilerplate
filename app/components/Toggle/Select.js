import styled from '@emotion/styled'

const Select = styled.select`
  line-height: 1em;
  background-color: transparent;
  border-style: none;
`

export default Select
