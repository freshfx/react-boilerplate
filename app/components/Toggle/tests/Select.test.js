import React from 'react'
import {render} from '@testing-library/react'

import Select from '../Select'

describe('Select', () => {
  it('should match snapshot', () => {
    const {container} = render(<Select />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
