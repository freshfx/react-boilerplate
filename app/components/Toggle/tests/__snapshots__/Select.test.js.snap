// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select should match snapshot 1`] = `
.c0 {
  line-height: 1em;
  height: 20px;
}

<select
  class="c0"
/>
`;
